app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesActivityTest.kt

```diff
@@ -132,6 +172,46 @@ class PoliciesActivityTest {
     }
   }
 
+  @Test
+  fun testPoliciesActivity_inTermsOfServicePage_clickOnPrivacyLink_callsRouteToPrivacyPolicy() {
+    launch<PoliciesActivity>(
+      PoliciesActivity.createPoliciesActivityIntent(
+        ApplicationProvider.getApplicationContext(),
+        PolicyPage.TERMS_OF_SERVICE
+      )
+    ).use { scenario ->
+      scenario.onActivity {
+        onView(withId(R.id.policy_description_text_view))
+          .check(matches(isDisplayed()))
+        // Verify the displayed text is correct & has a clickable span.
+        val textView = it.findViewById<TextView>(R.id.policy_description_text_view)
+        val spannableString = textView.text as Spannable
+        val clickableSpans = spannableString.getSpansFromWholeString(ClickableSpan::class)
+        clickableSpans.first().onClick(textView)
+        testCoroutineDispatchers.runCurrent()
+
+        val policiesArguments =
+          PoliciesActivityParams
+            .newBuilder()
+            .setPolicyPage(PolicyPage.PRIVACY_POLICY)
+            .build()
+        Intents.intended(
+          IntentMatchers.hasComponent(
+            ComponentNameMatchers.hasClassName(
```

Comment:
Please directly import these & fit this to one line as we do elsewhere.

Ditto for hasExtras below.

Reply:
Done.